bay_trimesh/anaconda-project.yml

```diff
@@ -0,0 +1,53 @@
+name: bay_trimesh
+description: Visualizing water depth into the Chesapeake and Delaware Bays
+maintainers:
+- philippjfr
+
+channels:
+- pyviz
+
+dependencies: &deps
+- colorcet ==2.0.1
+- datashader ==0.7.0
+- geoviews ==1.6.2
+- holoviews ==1.12.2
+- notebook ==5.7.8
+- pandas ==0.24.2
+
+packages: *deps
+
+platforms:
+- linux-64
+- osx-64
+- win-64
+
+commands:
+  notebook:
+    notebook: bay_trimesh.ipynb
+    env_spec: pyviz-topics-bay-trimesh
+  test:
+    unix: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    env_spec: test
```

Comment:
It's important to be able to specify commands per platform, but given they are the same here and will generally be the same, is there no way to specify the command just once?  It's also ambiguous what will happen on osx; is that considered unix?  If we do need to have both specs, please align them so that we can see at a glance that the commands are identical.

Reply:
I can use a reference if they are both needed. Let me check

Reply:
In this case with a single-line command, a reference is probably more confusing than having it copied, as long as the two copies are lined up and visually identical.  But if there's a way to specify it for all platforms as a default that can be overridden per platform, that's the way I'd rather do it in general.

Reply:
Ok so osx counts as unix and I believe you need to specify a command per platform. So I will align them visually.

Reply:
I am imagining that there will be some special anaconda-project init for these example projects and they will autopopulate this section.

Reply:
Ok, thanks.  Please normalize the spacing before "&deps" and "*deps" across the files; seems like it varies.  Probably just one space there is fine; it's the least surprising.